<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NLP_Pytorch_Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_PretrainedModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PretrainedModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>选择预训练模型根路径（默认是model）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PretrainedModel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>320</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>../model/</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Data">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>320</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>./data/</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Data">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>选择数据根路径（默认是data）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BestModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>选择模型保存根路径（默认是checkpoints）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BestModel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>320</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>./checkpoints/</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BestModel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TaskName">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>任务名称</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>920</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RunParse">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Train">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Train">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>训练进度</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Dev">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dev">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>评估进度</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>文件</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_PretrainedModel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse_PretrainedModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_BestModel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse_BestModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse_Data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_PretrainedModel()</slot>
  <slot>browse_Data()</slot>
  <slot>browse_BestModel()</slot>
  <slot>runparse()</slot>
 </slots>
</ui>
